src/reader/ArticleReader.js

```diff
@@ -304,15 +309,27 @@ export default function ArticleReader({ api, teacherArticleID }) {
         <div id={"bottomRow"}>
           <ReviewVocabulary articleID={articleID} />
           <s.CombinedBox>
-            <h4> {strings.answeringMsg} </h4>
+            <p style={{padding: "0em 2em 0em 2em"}}> {strings.answeringMsg} </p>
             <LikeFeedBackBox
               api={api}
               articleID={articleID}
               articleInfo={articleInfo}
```

Comment:
is `articleID` separately necessary? IIRC, `articleInfo.id` already has that info.

Reply:
Changed to only use articleInfo.id :)